scene/main/window.cpp

```diff
@@ -1212,6 +1212,15 @@ void Window::popup(const Rect2i &p_screen_rect) {
 
 	set_transient(true);
 	set_visible(true);
+
+	int screen_id = DisplayServer::get_singleton()->window_get_current_screen(get_window_id());
+	Rect2i screen_rect = Rect2i(DisplayServer::get_singleton()->screen_get_position(screen_id), DisplayServer::get_singleton()->screen_get_size(screen_id));
```

Comment:
```suggestion
	Rect2i screen_rect = DisplayServer::get_singleton()->screen_get_usable_rect(screen_id);
```
Not a big difference, but can also prevent popups from spawning under the taskbar.